src/components/Tooltip/Tooltip.test.tsx

```diff
@@ -21,10 +21,10 @@ const TooltipTest = (props: Partial<TooltipProps>) => {
       text='Tooltip text'
       offset={2}
       placement='bottom-start'
-      container={document.getElementById('portal-root')}
+      container={document.getElementById('portal-root') ?? undefined}
       {...props}
     >
-      <span>Testing</span>
+      <span data-testid='span'>Testing</span>
```

Comment:
as a test id I think this should be more specific. Something like "tooltip-inner-span"

Reply:
done! :heavy_check_mark: